app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+class IncorrectCommand(Exception):
+    """Error if command is not move"""
+
+
+def move_file(command: str) -> None:
+    command, copied_file, new_file = command.split()
+    if command != "mv":
+        raise IncorrectCommand("Function supports only mv command")
+
+    if "/" in new_file:
+        folders = new_file.split("/")[:-1]
+        file_path = [folders[0]]
+        for i in range(1, len(folders)):
+            file_path.append(f"{file_path[i - 1]}/{folders[i]}")
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220118765-d2989911-1a6d-411a-a4a2-baf16c348569.png)
fix this